ProcessMaker/processmaker
# Feature/FOUR-16933: infiniteScroll for mobiles tables and tabs

## How to Test
- Login to ProcessMaker from Mobile
- Go to Tasks and Cases
- If there is no more than 15 cases or tasks create almost 16
- Scroll down to the bottom of the screen
- Infinite scroll should work retrieving more rows

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-16933

## Code Review Checklist
ci:next
